src/NuGet.Core/NuGet.Commands/RestoreCommand/LockFileBuilder.cs

```diff
@@ -217,10 +214,10 @@ public class LockFileBuilder
                             library,
                             includeFlags,
                             targetGraph,
-                            rootProjectStyle,
-                            maccatalystFallback);
+                            rootProjectStyle);
 
                         target.Libraries.Add(projectLib);
+                        continue;
```

Comment:
Do we really need this one? It looks code works exactly same with or without this one.

Reply:
Yeah, it can be removed.